sonar-cxx-plugin/src/test/java/org/sonar/plugins/cxx/TestUtils.java

```diff
@@ -171,16 +172,16 @@ public static void addInputFile(DefaultFileSystem fs,
         temp.setCharAt(0, Character.toUpperCase(temp.charAt(0)));
         path = temp.toString();
       }
-      inputFile = new DefaultInputFile(path);
-      inputFile.setAbsolutePath(path);
+      inputFile = new DefaultInputFile("key", path);
```

Comment:
I don't know? Should we not trust the platform will do its job and import all files accordingly?

Probably all this file import could be simplified. And the good thing is that we don't have any maintenance when they decide to change all the filesystem interfaces as they did for 5.2